platform/contracts/timealarms/src/contract.rs

```diff
@@ -2,7 +2,7 @@ use platform::{
     batch::{Emit, Emitter},
     response,
 };
-#[cfg(feature = "cosmwasm-bindings")]
+#[cfg(feature = "contract")]
```

Comment:
Similarly to the Admin, the module is already under `cfg(feature = contract)`. Noticed that it's in also when `test` ... have doubts it should. I believe it was added because the module contains tests .... but they are relevant to the contract only, so should be executed only when the feature is `contract`